src/pages/home.tsx

```diff
@@ -1,24 +1,71 @@
 import {
+    HiArrowLeft,
+    HiArrowRight,
     HiArrowTopRightOnSquare,
     HiOutlineQuestionMarkCircle,
     HiPencil,
     HiUserCircle,
 } from "react-icons/hi2";
+import { useEffect, useState } from "react";
 import OpenSaucedLogo from "../assets/opensauced-logo.svg";
 import { useAuth } from "../hooks/useAuth";
 import { useOpensaucedUserCheck } from "../hooks/useOpensaucedUserCheck";
 import { Profile } from "./profile";
 import { goTo } from "react-chrome-extension-router";
 import AIPRDescription from "./aiprdescription";
 import PostOnHighlight from "./posthighlight";
-
+import { getHighlights } from "../utils/fetchOpenSaucedApiData";
 
 import Help from "./help";
 import { OPEN_SAUCED_INSIGHTS_DOMAIN } from "../constants";
+interface Highlight {
+    highlight: string;
+    title: string;
+    name: string;
+    url: string;
+}
+
 
 const Home = () => {
     const { user } = useAuth();
     const { currentTabIsOpensaucedUser, checkedUser } = useOpensaucedUserCheck();
+    const [highlights, setHighlights] = useState<Highlight[]>([]);
+    const [currentPage, setCurrentPage] = useState(0);
+    const [currentName, setCurrentName] = useState<string>("");
```

Comment:
GitHub says you aren't using this variable anywhere.

Reply:
```suggestion
```